starboard/shared/starboard/player/decoded_audio_internal.cc

```diff
@@ -73,6 +73,26 @@ DecodedAudio::DecodedAudio(int channels,
   //           0);
 }
 
+DecodedAudio::DecodedAudio(int channels,
+                           SbMediaAudioSampleType sample_type,
+                           SbMediaAudioFrameStorageType storage_type,
+                           SbTime timestamp,
+                           int size_in_bytes,
+                           Buffer&& storage)
+    : channels_(channels),
+      sample_type_(sample_type),
+      storage_type_(storage_type),
+      timestamp_(timestamp),
+      storage_(std::move(storage)),
+      offset_in_bytes_(0),
+      size_in_bytes_(size_in_bytes) {
+  SB_DCHECK(channels_ > 0);
+  SB_DCHECK(size_in_bytes_ >= 0);
+  // TODO(b/275199195): Enable the SB_DCHECK below.
```

Comment:
Should this check be enabled? The comment indicates it as a TODO.

Reply:
Remove this comment as it is enabled below.